File: pythonproject/service/estatistica_campi.py
from service.estatistica_base import EstatisticaBase
from model.Pessoa import Pessoa
from typing import List

class EstatisticaCampi(EstatisticaBase):
    CAMPUS = [
        "SAO CRISTOVAO", "ARACAJU", "ITABAIANA", "CAMPUS DO SERTAO", "LARANJEIRAS"
    ]

    def calcular(self) -> List[float]:
        return self.calcular_porcentagens(self.CAMPUS, Pessoa.get_campus)

File: javaproject/src/main/java/service/EstatisticaCampi.java
package service;

import model.*;
import java.util.List;

public class EstatisticaCampi extends EstatisticaBase {
    private static final String[] CAMPUS = {
        "SAO CRISTOVAO", "ARACAJU", "ITABAIANA", "CAMPUS DO SERTAO", "LARANJEIRAS"
    };

    @Override
    public List<Double> calcular() {
        return calcularPorcentagens(CAMPUS, Pessoa::getCampus);
    }
}